### DISCUSSION TITLE
Button to clear all entries

### DISCUSSION THREAD

**Discussion by SamuelNitsche** [spatie/ray#216] [Ideas]
As far as I know, the only way to clear all entries in ray is to clear it in the code.

I think it would be nice to clear entries using a button in the UI.
This way you could just rerun your code and have the output in a fresh, clean ray window.

**Comment by freekmurze** [comment]
In the latest release of Ray, we've added a menu item to do this! 
<img width="373" alt="image" src="https://user-images.githubusercontent.com/483853/105097420-d507dd00-5aa8-11eb-8733-5e54d6a9fa2f.png">

**Reply by SamuelNitsche** [reply]
You are a wizard 🧙‍♂️
